extensions/github-authentication/src/githubServer.ts

```diff
@@ -338,7 +342,23 @@ export class GitHubServer implements IGitHubServer {
 
 	private async doLoginWithPat(scopes: string): Promise<string> {
 		this._logger.info(`Trying to retrieve PAT... (${scopes})`);
-		const token = await vscode.window.showInputBox({ prompt: 'GitHub Personal Access Token', ignoreFocusOut: true });
+
+		const button = vscode.l10n.t('Continue to GitHub');
+		const modalResult = await vscode.window.showInformationMessage(
+			vscode.l10n.t('Continue to GitHub to create a Personal Access Token (PAT)'),
+			{
+				modal: true,
+				detail: vscode.l10n.t('To finish authenticating, navigate to GitHub to create a PAT then paste the PAT into the input box.')
+			}, button);
+
+		if (modalResult !== button) {
+			throw new Error('User Cancelled');
+		}
+
+		const description = `${vscode.env.appName} (${scopes})`;
+		const uriToOpen = await vscode.env.asExternalUri(this.baseUri.with({ path: '/settings/tokens/new', query: `description=${description}&scopes=${scopes.split(' ').join(',')}` }));
+		await vscode.env.openExternal(uriToOpen);
+		const token = await vscode.window.showInputBox({ placeHolder: `ghp_1a2b3c4...`, prompt: `GitHub Personal Access Token - ${scopes}`, ignoreFocusOut: true });
```

Comment:
The prompt needs localization?

Reply:
I decided against it because: "GitHub" shouldn't be localized and since GitHub itself isn't localized, I'm not sure it's worth the confusion of localizing "Personal Access Token" only in VS Code rather than having the association with the website which says "Personal Access Token".